src/modules/legacy/components/Status/Status.js

```diff
@@ -5,7 +5,7 @@ import statusMessages from './statusMessages';
 import styles from './status.css';
 
 const Status = ({
-  account, transactions, t,
+  account, transactions, t, prevStep
```

Comment:
I cannot see that we pass in `prevStep` as a prop to `<Status` ? And if we want to extend its API prevStep seems like a very specific name, maybe `onRetry` would be better?

Reply:
so prevStep is a function exposed by the `<MultiStep />` component to its children to navigate between child steps

Reply:
Where are we using this legacy status with our forms? The statuses in the multistep seems to be specific statuses components.